### ISSUE TITLE
Using ArcFace loss with facenet model

### ISSUE THREAD

**Issue by divyanshpal** [sirius-ai/MobileFaceNet_TF#23]
I am trying to implement arcface loss for facenet training.
I replaced cross_entropy_mean with arcface_mean which I got from arc face function.
The loss is around 40 and accuracy is 0.00 and not changing. The loss is increasing.
Any suggestion what I might be doing wrong?

**Comment by sirius-ai** [comment]
since my code didn't have cross_entropy_mean, i don't know where you modified, but cross_entropy_mean and arcface_mean aren't as same meaning.
close this issue, when need reopen.
